DirectX11Proj/ResourceManager.h

```diff
@@ -26,7 +28,6 @@ class ResourceManager
 		return instance;
 	}
 
-
 	std::vector<ModelClass*> LoadModels(std::string aFilePath);
```

Comment:
If people are loading in models dynamically throughout the gameplay, you may want to consider using std::list instead